PizzaBuilder/PizzaBuilder/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace PizzaBuilder
+{
+    class Program
+    {
+
+        static void Main(string[] args)
+        {
+            //client
+            var director = new Director();
+            var builder = new WhitePizzaBuilder();
```

Comment:
It would be better if you used polymorphism to define builders, without creating new builder variable for each one.